plugins/MultiSites/Dashboard.php

```diff
@@ -73,20 +80,20 @@ public function __construct($period, $date, $segment)
 
     public function setSitesTable(DataTable $sites)
     {
-        $this->numSites     = $sites->getRowsCount();
         $this->sitesByGroup = $this->moveSitesHavingAGroupIntoSubtables($sites);
+        $this->rememberNumberOfSites();
```

Comment:
This is another fix, before we did not count the found groups into `numSites` but groups were treated as sites. This is now fixed as well.